Examples/MuxStatsGoogleIMAPluginSPMExampleIOS/MuxStatsGoogleIMAPluginSPMExampleIOS/PlayerContainerViewController.swift

```diff
@@ -168,11 +198,12 @@ extension PlayerContainerViewController: IMAAdsLoaderDelegate {
         }
 
         loadedAdsManager.delegate = self
-        
+
         // MARK: Monitor the IMAAdsManager with Mux
-        // note - do this *after* setting your delegate but *before*
+        // note - do this *after* setting your delegate but
+        // not *before*
```

Comment:
```suggestion
        // note - do this *after* setting your delegate but
        // *before* calling `IMAAdsManager:initialize:with`
```
Not sure how it got cropped-off, but the comment should be this

Reply:
15c0f71